compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/tree/STSpreadMemberNode.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we not commit these changes in [STSpreadMemberNode.java](https://github.com/ballerina-platform/ballerina-lang/pull/36384/files#diff-cbf9399d326ccb6420176ff8ac193539c3f33d20d64532bd489d87f246951ad9) and [SpreadMemberNode.java](https://github.com/ballerina-platform/ballerina-lang/pull/36384/files#diff-e78f8d41e56e1ffd9c60e670288f203689a35b379f70211d1aae62ea7918f9cc).

This diff is because currently treegen does not support updating license header year.